requirements.txt

```diff
@@ -34,6 +34,7 @@ SQLAlchemy-Utils==0.34.2
 sqlparse==0.3.0
 statsd==3.3.0
 gunicorn==19.9.0
+greenlet==0.4.16
```

Comment:
What's this one for?

Reply:
Yea this probably should've been its own PR, but it's now needed to be specified explicitly due to a conflict in dependencies.